x/stablestake/keeper/debt.go

```diff
@@ -57,6 +57,12 @@ func (k Keeper) SetInterest(ctx sdk.Context, block uint64, interest types.Intere
 	}
 }
 
+func (k Keeper) SetInterestBlock(ctx sdk.Context, interest types.InterestBlock) {
```

Comment:
I am not if we need this function. also key is different on how we store interest block
we have `SetInterest`

Reply:
we use block height as key

Reply:
@amityadav0 this function is used for importing interest block objects to the genesis state of the module, an interest block already has the block  height (blockTime),

Reply:
`SetInterest` requires `block` as param which will not work for genesis import flow

Reply:
got it, looks like we need to migrate data to include `block_height` field and then add it in genesis

Reply:
@amityadav0 changes done, please review again thanks